src/RecoveryServices/RecoveryServices.Backup/Cmdlets/Item/InitializeAzureRmRecoveryServicesBackupProtectableItem.cs

```diff
@@ -76,8 +76,9 @@ public override void ExecuteCmdlet()
                 var operationStatus = TrackingHelpers.GetOperationResult(
                inquiryResponse,
                operationId =>
-                   ServiceClientAdapter.GetContainerRefreshOrInquiryOperationResult(
+                   ServiceClientAdapter.GetRegisterContainerOperationResult(
```

Comment:
@siddharth7 is the functionality for this covered in the test re-recorded above?

Reply:
Yes, TestAzureVmWorkloadInitializeProtectableItem.json contains the required changes